website/docs/d/container_app.html.markdown

```diff
@@ -283,6 +285,18 @@ A `custom_domain` block supports the following:
 
 ---
 
+A `ip_security_restriction` block supports the following:
```

Comment:
Can you please update all the other block descriptions in these docs to use `exports` instead of `supports`
```suggestion
A `ip_security_restriction` block exports the following:
```

Reply:
Fixed.